developer/ruby/export_openstudio_standards_libraries.rb

```diff
@@ -0,0 +1,548 @@
+# This file contains methods that export some of the contents of the
+# OpenStudio-Standards to library .osm files that are packaged
+# with the OpenStudio installer.
+# These library .osm files are not used by the OpenStudio-Standards gem,
+# but this export functionality is part of the gem because the
+# gem itself contains the content needed for the libraries.
+
+require 'json'
+require 'openstudio'
+require 'openstudio-standards'
+
+# gem install parallel
+require 'parallel'
+
+class FakeSqlFile
+end
+
+module OpenStudio
+module Model
+
+  class Model
+    def getAutosizedValue(object, str, units)
+      puts "OH YEAH"
+      STDOUT.flush
+
+      case str
+      when 'Design Size Rated Air Flow Rate'
+        return OpenStudio::OptionalDouble.new(1)
+      when 'Design Size Gross Rated Total Cooling Capacity'
+        return OpenStudio::OptionalDouble.new(1)
+      when 'Design Size Gross Rated Sensible Heat Ratio'
+        return OpenStudio::OptionalDouble.new(1)
+      end
+
+      super(object, str, units)
+    end
+
+    def sqlFile
+      return FakeSqlFile.new
+    end
+  end
+end
+end
```

Comment:
@macumber I've merged the branch since this works now. But I'm curious why you added that bit, and whether we couldn't at least remove the "OH YEAH" or something.

Reply:
This code in Standards requires a sizing run so it can find sizes.  I wanted to fake that out so we didn't have to do a sizing run to generate standards OSMs.  I just left in OH YEAH so I could see where it was being called.  It's just for debugging, can be removed